data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1617 @@
+			choice
+				`	"That is odd."`
+					goto movingon
+				`	"What did the Remnant get?"`
+					goto remnantgain
+				`	"What leverage did the Remnant use to bargain?"`
+			`	Gavriil smiles. "Well, it seems that the captain of that ship found it distressing that we could pinpoint and detail them and their crew with sufficient data that we could use their own appearance as fake video in our transmissions, not to mention that we successfully identified which of the people on board the ship were them. And it was doubly distressing for them that, despite their impressive scanner arrays, they were not able to even catch the slightest emission that might hint at where or who we were." He pauses as he palms open a door to pull the loaded lift through, then continues. "They blustered initially, but when we started listing off precise defects and wear present on their various computer systems they decided it was better to make a deal."`
+			choice
+				`	"Clever!"`
+					goto movingon
+				`	"What did you get from them?"`
+			label remnantgain
+			`	You finish unloading the equipment into the room before Gavriil continues. "Well, the Korath have been around for a comparatively long time. So we asked for their astronomical dataset. The data we received was raw, uncorrected, and continuous data covering multiple tens of thousands of years of stellar observations. The data even includes data that likely corresponds to the destruction of some of their worlds during their civil war." He pauses, then continues. "Of course, we need to verify the data first. So far, though, it is checking out successfully against trusted data. If it can be validated, the data will massively expand the dataset our researchers can use to model and study the universe."`
+			label movingon
+			`	Gavriil tucks the lifter away into a storage alcove. "Well, thank you for your help, <first>. That definitely sped up the process."`
+			label gavriilend
+			`	He looks around at the stacks of equipment, and turns to you. "This will keep me busy for a few days, but if you would be willing to do some patrols, it would be much appreciated. I will have details for you once I have a few things unpacked."`
+
+
+
+mission "Remnant: Cognizance 25"
+	name "Equipment to Ssil Vida"
```

Comment:
This is a duplicate of the last mission's name, needs to be updated.

Reply:
Resolved now. Thanks!